src/ir/constant-expr.cc

```diff
@@ -0,0 +1,166 @@
+#include "constant-expr.h"
+#include "type.h"
+
+
+NAN_MODULE_INIT(ConstantExprWrapper::Init) {
+    auto constantExpr = Nan::GetFunction(Nan::New(constantExprTemplate())).ToLocalChecked();
+
+    Nan::Set(target, Nan::New("ConstantExpr").ToLocalChecked(), constantExpr);
+}
+
+v8::Local<v8::Object> ConstantExprWrapper::of(llvm::ConstantExpr* constantExpr) {
+    auto constructor = Nan::GetFunction(Nan::New(constantExprTemplate())).ToLocalChecked();
```

Comment:
Please also add ConstantExpr to the `ConstantWrapper::of` operation. This ensures that instances of `ConstantExpr` are returned as such.

Reply:
ping @vihanb